### PR TITLE
feat: migrate local structs to `svc-storage`

### PR DESCRIPTION
**Pull request by GoodluckH**
## Changes
* Closes #5 
## Notes
* Hardcoded `created_at` and `updated_at` fields as these fields are not available on `svc-storage` at the moment
* Masks are expected to be generated by frontend clients.
 * For `status` field, all assets are set as `Available` as default. This `status` field doesn't have a counterpart on svc-storage as of yet.
 * `group_id` does not exist on storage for vertiports
 * `description` for vertiports are mandatory on storage but optional for local structs in svc-assets
 * Asset groups and delegations are ignored as they are not the focus of R2

### CODE REVIEW

**Review by amsmith-pro — DISMISSED** [review]

File: server/Cargo.toml
Comment (amsmith-pro) [inline server/Cargo.toml]: I think it's actually better to use the specific tags, for example there was a big merge into svc-storage just now that will break this PR if you depend on develop

File: server/src/grpc_clients.rs
Comment (amsmith-pro) [inline server/src/grpc_clients.rs]: can use a macro here as these are fairly repetitive:
https://github.com/Arrow-air/svc-telemetry/blob/d2a1695c5999c1e8a7f6a36e9c6d7e1b0ca1efc2/server/src/grpc_clients.rs#L78

**Review by amsmith-pro — CHANGES_REQUESTED** [review]

File: server/Cargo.toml
Comment (GoodluckH) [reply server/Cargo.toml]: [92165fa](https://github.com/Arrow-air/svc-assets/pull/7/commits/92165faff0a4ae8b2c5d9e09519ab604d31d2a10)

File: server/src/grpc_clients.rs
Comment (GoodluckH) [reply server/src/grpc_clients.rs]: [92165fa](https://github.com/Arrow-air/svc-assets/pull/7/commits/92165faff0a4ae8b2c5d9e09519ab604d31d2a10)

**Review by amsmith-pro — APPROVED** [review]

**Comment by amsmith-pro** [comment]
I'll let @owlot have final signoff on this

**Review by owlot — APPROVED** [review]

**Pull request merged**
